### PR TITLE
Adding separate iOS & Android markdown files & (hopefully) finishing up client side setup

### PR DESCRIPTION
**Pull request by adilanchian**
Looking for some feedback on the changes I made to the README! Currently working on getting the file Backend README setup as well. Any and all suggestions are welcome :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,76 +20,42 @@
...
+If you are interested in becoming a member of the team check out the **[PixelogicDev Twitch](https://twitch.tv/pixelogicdev)**, the **[PixelogicDev Discord](https://discord.gg/8NFtvp5)** and **[contribute](README-Support/CONTRIBUTING.md)** to this awesome project!
 
-# MVP Checklist & Timeline For Beta
+# Release Checklist
 
-### Checklist
+This living list of items is something that will constantly be updated as we develop through Grüvee.
 
-#### Authentication
+-   [x] Releae Alpha to community members (iOS/Android)
```
Comment (isabellabrookes) [inline README.md:37]: Typo: Realae ===> Release

File: README.md [hunk 2]
```diff
@@ -125,21 +91,66 @@ In order to develop for Grüvee you are going to need to have a few things insta
...
+Now we are at the GOOD stuff. Time to get Grüvee running locally. Make sure you have the [prerequisites](#prerequisites) installed. Open up **two instances** of your favorite terminal/command prompt and navigate to the root directory of the Grüvee repository for both of them. One is going to run your React Native bundle server and the other will start the Grüvee iOS or Android app.
 
-## TODO - Add APK build process
+## 🌲 Environment Variables
 
--   `./gradlew app:assembleRelease`
+We have included an [`.example.env`](/.example.env) that has the properties that are needed for the app to run properly. Start here by making a `.env` in the root of the repo
 
-# Running Grüvee Locally
+### `ENVIRONMENT`
```
Comment (isabellabrookes) [inline README.md:110]: I think you could probably put the environmental variable directions inside the `README-Support/` section.

**Comment by isabellabrookes** [comment]
The more I look at the structure of the README.md, the more I wonder if we need to break it up. 
Unrefined thoughts: 

1. **README.md**

- Explains the app, screenshots/videos/gifs/etc (BRING THE HYPE)
- Pixelogic community stressed

2. **README-Support/**
a. Setting up
b. Contributing
c. Roadmap/todos

**Comment by adilanchian** [comment]
@isabellabrookes - I like your thought process here. We should showcase the app a bit more! Going to make same changes.

**Comment by isabellabrookes** [comment]
Test

@all-contributors please add @adilanchian for code, docs

**Pull request merged**